docs/README.md

```diff
@@ -0,0 +1,69 @@
+# Kite for Visual Studio Code
+
+This extensions adds support for Kite in Visual Studio Code, including:
+
+- Completion Lists
+- Signature Help
+- Quick Info
+- Goto Definition 
+
+## How to use this extension
```

Comment:
Change this to:
```
## How to install this extension
```
Once this is in the marketplace, we can update it with instructions about installing the Kite executable via the plugin